weave/trace/custom_objs.py

```diff
@@ -9,6 +9,19 @@
 from weave.trace.serializer import get_serializer_by_id, get_serializer_for_obj
 
 
+class DecodeCustomObjectError(Exception):
+    """An error that occurs while decoding a custom object."""
+
+
+# in future, could generalize as
+# {target_cls.__module__}.{target_cls.__qualname__}
+KNOWN_TYPES = {
+    "Op",
```

Comment:
does this work for Ops? I thought ops were a special hard coded one? I thought at one time we had special Op handling in `def from_json(obj: Any, project_id: str, server: TraceServerInterface) -> Any:`

Reply:
I stand corrected. nvm